src/crimson/common/errorator.h

```diff
@@ -481,7 +482,7 @@ struct errorator {
       return this->then_wrapped(
         [ valfunc = std::forward<ValueFuncT>(valfunc),
           errfunc = std::forward<ErrorVisitorT>(errfunc)
-        ] (auto&& future) mutable [[gnu::always_inline]] noexcept {
```

Comment:
could you revise the title of the commit message?

instead of 

> CLANG-related fix to errorator.h

might want to add a prefix to specify the component which is changed by this commit. and use the imperative mood in the title. see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#commit-title. in this case it could be

> crimson: put noexcept specifier after attributes specifiers.

and could reference the section of [dcl.fct] in N4713. and s/re/regarding/

> D1 ( parameter-declaration-clause ) cv-qualifier-seq_{opt}
>    ref-qualifier_{opt} noexcept-specifier_{opt} attribute-specifier-seq_{opt}

Reply:
done

Reply:
done